### PR TITLE
Chromium-only article migrations

### PR DESCRIPTION
**Pull request by AaronForinton**
Linked to https://github.com/GoogleChrome/web.dev/pull/9601

**Ready for review**

### CODE REVIEW

**Comment by AaronForinton** [comment]
@rachelandrew Do you recognise the build error here?

File: site/en/articles/declarative-shadow-dom/index.md [hunk 1]
```diff
@@ -0,0 +1,386 @@
...
+authors:
+  - developit
+  - masonfreed
+description: |
+  Declarative Shadow DOM is a new way to implement and use Shadow DOM directly in HTML.
+tags:
+  - dom
+  - html
+  - javascript
+  - layout
```
Comment (rachelandrew) [inline site/en/articles/declarative-shadow-dom/index.md:19]:
```suggestion
```

**Comment by rachelandrew** [comment]
It's the tags, for example `i18n.tags.network`, the network tag must be missing.

**Comment by matthiasrohmer** [comment]
This PR had some build errors, that I fixed with the following commits. They are probably worth looking out for in upcoming migrations.

- The {% YouTube %} shortcode for developer.chrome.com expects named arguments, see https://github.com/GoogleChrome/developer.chrome.com/pull/5326/commits/2d2dc89d284a01200916b5bf4e320a0c7f5bdbed - web.dev is less strict on this. Filed https://github.com/GoogleChrome/webdev-infra/issues/50 to fix this.
- Missing authors need to be added to both authorsData.json and authors.yaml, see https://github.com/GoogleChrome/developer.chrome.com/pull/5326/commits/6c3c2f1a2fb4ca581f1d351a5cfae8e69826b976
- d.c.c enforces `alt` textes for hero images, see https://github.com/GoogleChrome/developer.chrome.com/pull/5326/commits/307e1a7ca66e6f841ee5daa8d25a0cd64576c85f
- The Declarative Shadow DOM article had broken syntax (corrupt closing tag) causing an unspecified `Unexpected end of document` error. The original document for web.dev doesn't have that issue, so that probably slipped in during editing

**Comment by rachelandrew** [comment]
Thanks @AaronForinton I've just asked the author of the Shadow DOM post to check that one as it was heavily rewritten.

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**